x-pack/plugins/secops/public/graphql/types.ts

```diff
@@ -156,7 +156,7 @@ export interface SourceEcsFields {
 
   port?: number | null;
 
-  domain?: string | null;
+  domain?: (string | null)[] | null;
```

Comment:
Is it possible for the domain array to actually contain a null? Looking at a few different queries makes it seem as if it's always either `null`, an empty array `[]`, or an array of just strings.

Reply:
Accidentally commented on the generated types -- sorry! Source schema for this is [here](https://github.com/elastic/kibana/pull/31836/files#diff-6cdc169d7ba43429ccde58738ffe9385R69). If it turns out that it'll never have `null` in the array the updated type would be `[String!]`. `domain` itself _can be_ null though, so you won't want to use `[String!]!`